tdrs-backend/tdpservice/settings/production.py

```diff
@@ -20,13 +19,11 @@ class Production(Common):
     INSTALLED_APPS += ("storages",)
     DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
     STATICFILES_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
-    AWS_ACCESS_KEY_ID = os.getenv("DJANGO_AWS_ACCESS_KEY_ID")
-    AWS_SECRET_ACCESS_KEY = os.getenv("DJANGO_AWS_SECRET_ACCESS_KEY")
-    AWS_STORAGE_BUCKET_NAME = os.getenv("DJANGO_AWS_STORAGE_BUCKET_NAME")
+
     AWS_DEFAULT_ACL = "public-read"
     AWS_AUTO_CREATE_BUCKET = True
     AWS_QUERYSTRING_AUTH = False
-    MEDIA_URL = f"https://s3.amazonaws.com/{AWS_STORAGE_BUCKET_NAME}/"
+    MEDIA_URL = f"https://s3.amazonaws.com/{Common.AWS_STORAGE_BUCKET_NAME}/"
```

Comment:
This was in here from before, this would result in the same url as was previously in there. We don't even use it I don't think, if I was going to change it I would remove it.

Reply:
I believe it's used by Django Admin for the other storage bucket